homeassistant/components/alexa/handlers.py

```diff
@@ -232,7 +232,27 @@ async def async_api_turn_off(
 
     service = SERVICE_TURN_OFF
     if entity.domain == cover.DOMAIN:
-        service = cover.SERVICE_CLOSE_COVER
+        open_and_close = cover.CoverEntityFeature.OPEN | cover.CoverEntityFeature.CLOSE
+
+        supported = entity.attributes.get(ATTR_SUPPORTED_FEATURES, 0)
+
+        # Only actually use the stop_cover service if the following conditions are met:
+        # AND:
+        #  - the cover has the STOP feature
+        #  - OR:
+        #    - the cover has open AND close
+        #    - the cover has set_position
+        #
+        # This ensures that the cover does not get stuck (although calling close for a cover
+        # which does not support closing could still result in an error)
+        # See https://github.com/home-assistant/core/pull/125851
```

Comment:
May be explain that Alexa translates a `stop` to power off the `AlexaPowerController`?

Reply:
What happens if a user wants to close the blinds instead, we can only assume we want to `stop` it is the state is `closing` or `opening`, so should we not check the state as well?